sql/sql_yacc.yy

```diff
@@ -12619,35 +12619,35 @@ show_param:
         | CLIENT_STATS_SYM opt_wild_or_where
           {
            LEX *lex= Lex;
-           Lex->sql_command= SQLCOM_SELECT;
+           Lex->sql_command= SQLCOM_SHOW_PLUGINS;
```

Comment:
Why? (Any discussion previously on Slack needs to be here and in the commit message later)

Reply:
"SQLCOM_SELECT implementation in sql_parse.cc is rewritten in 8.0." - why the rewritten implementation no longer works for userstat? And why SQLCOM_SHOW_PLUGINS` is a good replacement?